### PR TITLE
feat: slider and conclusion

### PR DESCRIPTION
**Pull request by SuyashTiwariCoder**
added: slider component and conclusion page .

### CODE REVIEW

File: Utilities/Conclusion/conclusion.html [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+        <div class="sidebar">
+  
+          <h3>Get Started</h3>
+          <div class="divider"></div>
+          <ul>
+            <li><a href="/Documentation/documentation.html">Introduction</a></li>
+            <li><a href="/Utilities/Colors/color.html">Colors</a></li>
+            <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+            <li><a href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+
```
Comment (ankit9015) [inline Utilities/Conclusion/conclusion.html:35]: Remove unused lines

File: Utilities/Conclusion/conclusion.html [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+            <li><a href="/components/Alert/alert.html">Alert</a></li>
+            <li><a href="/components/Button/button.html">Button</a></li>
+            <li><a href="/components/card/card.html">Card </a></li>
+            <li><a href="/components/Images Component/images.html">Image</a></li>
+            <li><a href="/components/Input/input.html">Input</a></li>
+            <li><a href="/components/modals/modal.html">Modal</a></li>
+            <li><a href="/components/Navigation/navigation.html">Navigation</a></li>
+            <li><a href="/components/Ratings/rating.html">Rating</a></li>
+            <li><a class="sidebar--active"href="/components/Snackbar/snackbar.html">Snackbar</a></li>
+
```
Comment (ankit9015) [inline Utilities/Conclusion/conclusion.html:53]: Here again

**Review by ankit9015 — APPROVED** [review]
Code looks good to merge

File: Utilities/Conclusion/conclusion.html (hunk 1 above)
Comment (SuyashTiwariCoder) [reply Utilities/Conclusion/conclusion.html:35]: Its not unused I have used it in ui . I have not connected it right now , I wil connect it in next pr

File: Utilities/Conclusion/conclusion.html [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+            <li><a href="/Documentation/documentation.html">Introduction</a></li>
+            <li><a href="/Utilities/Colors/color.html">Colors</a></li>
+            <li><a href="/Utilities/Text Utility/textUtil.html">Text Utility </a></li>
+            <li><a href="/Utilities/Conclusion/conclusion.html">Conclusion</a></li>
+            
+          
+          
+        </ul>
+          <h3>Components</h3>
+          <div class="divider"></div>
```
Comment (ganesh-kanchi) [inline Utilities/Conclusion/conclusion.html:40]: you could have used <hr> (hr) tag here, right?

**Review by ganesh-kanchi — APPROVED** [review]
good to be merged.

File: Utilities/Conclusion/conclusion.html (hunk 3 above)
Comment (SuyashTiwariCoder) [reply Utilities/Conclusion/conclusion.html:40]: Ganesh Ji which tag are you saying I am not able to understand . Can You be more specific?

**Pull request merged**